File: Samples/Android-Simple/app/src/main/java/com/pushwoosh/sample/quickstart/MainActivity.java
package com.pushwoosh.sample.quickstart;

import android.os.Bundle;

import com.pushwoosh.Pushwoosh;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_main);

		Pushwoosh.getInstance().registerForPushNotifications();
	}
}

File: Samples/Android-Advanced-Kotlin/app/src/main/java/com/pushwoosh/sample/MainActivity.kt
package com.pushwoosh.sample

import android.app.Activity
import android.os.Bundle
import android.util.Log

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Log.d(PushwooshSampleApp.LTAG, "MainActivity")
    }
}
